src/app.ts

```diff
@@ -11,6 +12,9 @@ app.get('/', (req, res) => {
 app.use(express.json());
 app.use(cors());
 
+// Add the user posts routes
+app.use('/api', userPostsRouter);
```

Comment:
Se sugiere que sea una ruta general
```suggestion
app.use('/api/posts', postsRouter);
```